entity-service-change-event-generator/src/main/java/org/hypertrace/entity/service/change/event/metric/EntityMetricsRegistry.java

```diff
@@ -0,0 +1,88 @@
+package org.hypertrace.entity.service.change.event.metric;
+
+import io.micrometer.core.instrument.Counter;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.AllArgsConstructor;
+import org.hypertrace.core.serviceframework.metrics.PlatformMetricsRegistry;
+
+public class EntityMetricsRegistry {
```

Comment:
This isn't actually a metric registry - it's just some accessors for well known metrics. please rename